packages/dapp/src/components/utility/Tabs.tsx

```diff
@@ -1,17 +1,32 @@
 import * as React from "react";
```

Comment:
Should these components be in `components` instead of `dapp`, so the tab functionality is shared and also testable in Storybook?